src/main/resources/db/migration/V20170713134558340__create_item_sequence.sql

```diff
@@ -0,0 +1,24 @@
+DELIMITER //
+CREATE PROCEDURE CreateSequence (`name` VARCHAR(30), `start` INT, `inc` INT)
+  BEGIN
+    CREATE TABLE IF NOT EXISTS `_sequences`
```

Comment:
It's a good point.  I think I am missing a drop statement on the stored proc and func.  And I need to be more defensive when inserting the sequence as well.

It is preferable to create idemponent scripts which I do want to shoot for.  I am not doing that in the first script I created because I didn't know how to do it with MySQL.  And it looks  like I'm maybe half way there in this script. I'll fix this.

Typically DDL scripts when creating tables, etc. would do a "IF NOT EXISTS ...".  MySQL does not support IF outside of stored procedures, so I've noticed a lot of MySQL scripts create tables inside a stored proc.  Typically the script will drop the stored proc first or last in the so it isn't kept around.

I probably don't need the CreateSequence to do the actual creating.  It seemed cool, but it would be better to do what I said -> stored proc to create the table if it doesn't exist, an insert of the actual sequence wanting to be created, then drop stored proc.